activesupport/lib/active_support/cache.rb

```diff
@@ -1021,6 +1047,7 @@ def initialize(value, compressed: false, version: nil, expires_in: nil, expires_
       end
 
       def value
+        @value = @value.call if @value.instance_of?(Proc)
```

Comment:
Why do we need to do this?  It's not obvious to me from the PR (but this PR is kind of long) 😅

(nm, I found it below)